roles/splunk_common/tasks/install_splunk_yum.yml

```diff
@@ -1,7 +1,12 @@
+# added the environment section due to bug:
+# https://github.com/ansible/ansible/issues/18979
```

Comment:
cc @hendolim - wondering if this might also apply to https://github.com/splunk/docker-splunk/issues/303